localization/english/culture/wc_cultural_maa_innovations_l_english.yml

```diff
@@ -173,7 +173,7 @@
  #####################
  
  innovation_blackrock_legionnaire:0 "Blackrock Legionnaires"
- innovation_blackrock_legionnaire_desc:0 ""
+ innovation_blackrock_legionnaire_desc:0 "The legionnaires are extremely strong, weilding very effective and and damaging weaponry.  Compared to other orcs, the legionnaires are highly disciplined and can work cohesively as a unit."
```

Comment:
if any, efficient and not effective afaik...

Reply:
not sure what you mean here

Reply:
https://dictionary.cambridge.org/de/grammatik/britisch-grammatik/efficient-or-effective